.github/workflows/push-tests.yml

```diff
@@ -128,6 +128,38 @@ jobs:
           # Build backend
           docker buildx build --cache-from=type=registry,ref=${{ secrets.ECR_REPO }}/genepi-backend:branch-trunk -t ${{ secrets.ECR_REPO }}/genepi-backend:latest src/backend
           make backend-check-style
+  e2e-test:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Configure AWS prod Credentials
+        uses: aws-actions/configure-aws-credentials@v1
+        with:
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-west-2
+          role-to-assume: ${{ secrets.AWS_ROLE_TO_ASSUME }}
+          role-duration-seconds: 900
+      - name: Login to ECR
+        uses: docker/login-action@v1
+        with:
+          registry: ${{ secrets.ECR_REPO }}
+      - uses: actions/checkout@v2
+      - name: Playwright tests
+        run: |
+          echo "DOCKER_REPO=${DOCKER_REPO}" > .env.ecr
+          # Build backend image
+          docker buildx build --cache-from=type=registry,ref=${{ secrets.ECR_REPO }}/genepi-backend:branch-trunk -t ${{ secrets.ECR_REPO }}/genepi-backend:latest src/backend
+          # Build frontend
+          docker buildx build --cache-from=type=registry,ref=${{ secrets.ECR_REPO }}/genepi-frontend:branch-trunk -t ${{ secrets.ECR_REPO }}/genepi-frontend:latest src/frontend
+          mkdir -p src/frontend/build
+          chmod a+rwx src/frontend/build # TODO temp hack - need a better plan for the build dir.
```

Comment:
We need to make sure the node user can write to the build directory, and file permissions work differently in docker-for-mac vs docker-for-linux.

Reply:
oh fun!